cache/s3proxy/s3proxy.go

```diff
@@ -196,6 +196,19 @@ func (c *s3Cache) Put(ctx context.Context, kind cache.EntryKind, hash string, si
 	}
 }
 
+func (c *s3Cache) CopyObject(ctx context.Context, bucket string, object string) {
+	_, err := c.mcore.CopyObject(
+		context.Background(),
+		bucket,
+		object,
+		bucket,
+		object,
```

Comment:
What's the 
```
		bucket,
 		object,
 		bucket,
 		object,
```
"dance" for ?

Reply:
it's for the 
`func (c Core) CopyObject(ctx context.Context, sourceBucket, sourceObject, destBucket, destObject string, metadata map[string]string, srcOpts CopySrcOptions, dstOpts PutObjectOptions) (ObjectInfo, error) {
`
we are moving it so same spot so that's why it's duplicated